applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -179,17 +179,27 @@ Check and configure the following configuration options for the sample:
 
    This option enables additional AT commands for using the SMS service.
 
-.. option:: CONFIG_SLM_GPS - GPS support in SLM
+.. option:: CONFIG_SLM_GNSS - GNSS support in SLM
 
-   This option enables additional AT commands for using GPS service.
+   This option enables additional AT commands for using GNSS service.
 
-.. option:: CONFIG_SLM_SUPL_SERVER - SUPL server
+.. option:: CONFIG_SLM_AGPS - nRF Cloud A-GPS support in SLM
 
-   This option specifies the SUPL server to use for retrieving SUPL A-GPS data.
+   This option enables additional AT commands for using nRF Cloud A-GPS service.
 
-.. option:: CONFIG_SLM_SUPL_PORT - SUPL server port
+   This option is not selected by default.
```

Comment:
You can have this right on the next line, without a blank line. -> "It is not selected by default.

Reply:
The empty line and sentence is the same as previous CONFIG items.
Remove all empty lines but keep the sentence no change to align with other CONFIG items not in this PR.